hips/utils/wcs.py

```diff
@@ -124,6 +124,30 @@ def create(cls, skydir: SkyCoord, shape: tuple, coordsys: str = 'CEL',
 
         return cls(w, shape)
 
+    @classmethod
+    def create_simple(cls, skydir: SkyCoord, shape: tuple, fov_deg: float,
+                      coordsys: str = 'CEL', projection: str = 'AIT') -> 'WCSGeometry':
```

Comment:
Yes.

I would suggest the end-user API of the hips package always has "icrs" and "galactic", and where needed we convert to other strings under the hood (e.g. to compare with `hips_frame`).